.pre-commit/run_sdk_checks.sh

```diff
@@ -1,41 +1,27 @@
 #!/usr/bin/env bash
 
-echo "starting sdk checks in dir: $(pwd)"
+set -euo pipefail
 
-source scripts/install_pnpm.sh
+CHANGED=$(git diff --name-only --cached)
+FILES=$(echo "$CHANGED" | grep '^sdk/' || true)
 
-# get dir
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+if [ -z "$FILES" ]; then
+  echo "no sdk changes, skipping SDK check..."
+  exit 0
+fi
 
-echo "SCRIPT_DIR used to jump to SCRIPT_DIR/../sdk =: $(SCRIPT_DIR)"
+echo "Running Biome on SDK files..."
 
-# nav to sdk dir
+# nav to sdk
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 SDK_DIR="$SCRIPT_DIR/../sdk"
-
-# get changed TS files
-TS_FILES=$(git diff --cached --name-only --diff-filter=ACMR | grep "^sdk.*\.\(ts\|tsx\)$" || true)
-
-echo "Running SDK checks..."
 cd "$SDK_DIR"
 
+# install deps
 echo "Installing deps via pnpm..."
 pnpm install
 
 # clean + build
-echo "Building SDK..."
 pnpm build
 
-# precommit checks (Biome)
-echo "Running checks..."
-pnpm check
-
-# CHECK_EXIT_CODE=$?
-
-# if [ $CHECK_EXIT_CODE -ne 0 ]; then
-#     echo "SDK check failed with $CHECK_EXIT_CODE"
-#     exit $CHECK_EXIT_CODE
-# fi
-
-# echo "SDK check completed..."
-
-exit 0
+pnpm pnpm check
```

Comment:
This should be `pnpm run` no?
```suggestion
pnpm run check
```